godotproject/Player.tscn

```diff
@@ -475,6 +475,7 @@ process_mode = 0
 one_shot = true
 
 [node name="Camera2D" type="Camera2D" parent="."]
+pause_mode = 2
```

Comment:
We make the camera always process even when the player is paused - this fixes a broken camera on level fade-in.